core/src/main/resources/map-pools.yml

```diff
@@ -26,6 +26,28 @@ pools:
     # How long should should a map cycle last?
     cycle-time: "15s"
 
+
+    # Voted pools support modifiers which come in the form of a formula (does not affect any other type of pool).
+    #
+    # This formula is parsed by exp4j library, it's quite flexible and supports many built-in functions.
+    #
+    # On top of that PGM implements:
+    # - bound(val, min, max)
+    #
+    # Available variables:
+    #  - score: the maps' score, determined by percentage of players who voted for it, or a default
+    #  - same_gamemode: amount of other maps of the same gamemode already picked for the vote
+    #  - mapsize: the size for this map in total players (sum of all team counts)
+    #  - players: amount of active players online (eg: obs count as half)
+    modifier: >-
```

Comment:
Should this be called `formula`?

Reply:
I think modifier is more accurate, it's not just a *formula* it's a *modifier* applied to the score to get what the weight for the map should be